plugins/modules/identity/ipa/ipa_otptoken.py

```diff
@@ -0,0 +1,529 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ipa_otptoken
+author: justchris1 (@justchris1)
+short_description: Manage FreeIPA OTPs
+description:
+- Add, modify, and delete One Time Passwords in IPA.
+options:
+  uniqueid:
+    description: Unique ID of the token in IPA.
+    required: True
+    aliases: ["name"]
+    type: str
+  newuniqueid:
+    description:  If specified, the unique id specified will be changed to this.
+    type: str
+  otptype:
+    description:
+    - Type of OTP.
+    - NOTE, Cannot be modified after OTP is created.
+    type: str
+    choices: [ totp, hotp ]
+  secretkey:
+    description:
+    - Token secret (Base64).
+    - If OTP is created and this is not specified, a random secret will be generated by IPA.
+    - NOTE, Cannot be modified after OTP is created.
+    type: str
+  description:
+    description: Description of the token (informational only).
+    type: str
+  owner:
+    description:  Assigned user of the token.
+    type: str
+  enabled:
+    description: Mark the token as enabled (default True).
+    default: True
```

Comment:
```suggestion
    description: Mark the token as enabled (default C(true)).
    default: true
```
Always use `true`/`false` in YAML, not the other variants.